lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -197,6 +162,12 @@ def _import_genbank_mass(self, params):
             # parse genbank file
             self._parse_genbank(genome_obj)
 
+            # check features
+            self.gi.check_dna_sequence_in_features(genome_obj.genome_data)
+
+            # validate genome
+            genome_obj.genome_data['warnings'] = self.gi.validate_genome(genome_obj.genome_data)
+
```

Comment:
Are there tests for G2G that exercise these code paths?

Reply:
Yeah, we have genbank_upload_full_test.py.